### ISSUE TITLE
Prototype state

### ISSUE THREAD

**Issue by alexhultman** [uNetworking/uTT#8]
If there was any misunderstanding, this repo is currently in prototype state. That means it has only been developed far enough to pass the benchmark so that I could decide if it was worth investing time in or not.

It will be fixed up as a v0.0.1 release within time but currently it is *completely broken* for production use. This is a base to build on, not a finished tower. The code is an absolute mess, so no need in reporting styling issues and such (yet).

Thanks

**Comment by StarpTech** [comment]
Why you don't mention this in the readme?

**Comment by alexhultman** [comment]
I do if you read.

**Comment by StarpTech** [comment]
Yes, I found it but it's confusing to find this in the "compilation" section. Thanks.
